### PR TITLE
Expand Sponsorship page to include JavaScript versions.

### PR DESCRIPTION
**Pull request by Shaptic**
Relevant live page is [here](https://developers.stellar.org/docs/glossary/sponsored-reserves/#examples).

### CODE REVIEW

**Comment by rice2000** [comment]
This looks good to me, but I can't verify the JS with 100% confidence. @tyvdh are you able to do that?

File: content/docs/glossary/sponsored-reserves.mdx [hunk 1]
```diff
@@ -129,6 +167,70 @@ Now, let's sponsor trustlines for Account A. Notice how the `CHANGE_TRUST` opera
 
 <CodeExample>
 
+```js
+  //
+  // 1. S1 will sponsor a trustline for Account A.
+  //
+  let s1Account = await server.loadAccount(S1.publicKey())
```
Comment (marcelosalloum) [inline content/docs/glossary/sponsored-reserves.mdx:174]: You could catch the error here too.
```suggestion
  let s1Account = await server.loadAccount(S1.publicKey()).catch(function (error) {
       console.error("  failed:", error.response.body);
     });
```

File: content/docs/glossary/sponsored-reserves.mdx [hunk 2]
```diff
@@ -129,6 +167,70 @@ Now, let's sponsor trustlines for Account A. Notice how the `CHANGE_TRUST` opera
...
+  let tx = new sdk.TransactionBuilder(s1Account, {fee: sdk.BASE_FEE})
+    .addOperation(sdk.Operation.beginSponsoringFutureReserves({
+      sponsoredId: A.publicKey(),
+    }))
+    .addOperation(sdk.Operation.changeTrust({
+      source: A.publicKey(),
+      asset: assets[0],
+      limit: "1000",
```
Comment (marcelosalloum) [inline content/docs/glossary/sponsored-reserves.mdx:182]: nit:
```suggestion
      limit: "1000",  // this limit can vary according with your application. If left empty it defaults to the max limit.
```

**Review by marcelosalloum — APPROVED** [review]
LGTM! I just left a couple of suggestions below ⬇️

**Comment by Shaptic** [comment]
Addressed in 754388fe63cf1dce6285732f0a7a030e2b21207c! @rice2000 can you :two: :heavy_multiplication_x:  :heavy_check_mark: and merge?

**Comment by Shaptic** [comment]
Just needed to add a few minor fixes. As a reminder, the live preview page is [here](http://developers-pr300.previews.kube001.services.stellar-ops.com/docs/glossary/sponsored-reserves/#examples).

**Review by rice2000 — APPROVED** [review]
Well done!  Thanks @Shaptic!

**Pull request merged**